cpp/Platform.Equality.Tests/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 3.8)
+project(Platform.Equality)
+
+include_directories(../Platform.Equality)
+
+include(FetchContent)
+FetchContent_Declare(
+        googletest
```

Comment:
It's not cool to download lib in cmake ;(. GTest can be easily [found](https://cmake.org/cmake/help/v3.20/module/FindGTest.html?highlight=gtest) instead:
```cmake
find_package(GTest REQUIRED)
# ...
target_link_libraries(Platform.Equality.Tests GTest::gtest GTest::gtest_main)
```

Reply:
We also can use `vcpkg` to make `find_package` work: https://vcpkg.readthedocs.io/en/latest/examples/installing-and-using-packages/#cmake-toolchain-file